pkgs/development/libraries/libcsptr/default.nix

```diff
@@ -0,0 +1,22 @@
+{ stdenv, fetchFromGitHub, cmake, git }:
+
+stdenv.mkDerivation rec {
+  name = "libcsptr-${version}";
+  version = "2.0.4";
+
+  src = fetchFromGitHub {
+    owner = "Snaipe";
+    repo = "libcsptr";
+    rev = "v${version}";
+    sha256 = "0i1498h2i6zq3fn3zf3iw7glv6brn597165hnibgwccqa8sh3ich";
+  };
+
+  nativeBuildInputs = [ cmake git ];
+
+  meta = with stdenv.lib; {
+    description = "This project is an attempt to bring smart pointer constructs to the (GNU) C programming language.";
```

Comment:
Please use a `description` that conforms to https://nixos.org/nixpkgs/manual/#sec-standard-meta-attributes.

Reply:
done